packages/harmony/src/storybook/components/ComponentRule.tsx

```diff
@@ -68,6 +68,7 @@ export const ComponentRule = (props: ComponentRuleProps) => {
           border: `1px solid ${borderColor}`,
           borderRadius: cornerRadius.m,
           boxSizing: 'content-box',
+          overflow: 'hidden',
```

Comment:
thoughts on `overflow: hidden` for all Flex and Box?

Reply:
i personally don't think it should be a default style, but thats just my opinion

Reply:
Yeah, it's pretty opinionated. I added it to paper because I think that always wants to contain its content, but I'm cool with leaving it off flex and box